app/assets/javascripts/application.js

```diff
@@ -57,10 +57,8 @@ $(document).ready(function(){
         dataIdArray.push($(this).attr("data-id"))
       }
     });
-    
 
     //console.log(dataIdArray);
```

Comment:
I realize that this line is not part of this PR. But, it's best not to leave debug comments around in your code. 😉